java/src/main/kotlin/io/spine/validation/java/JavaRendererExts.kt

```diff
@@ -33,11 +33,10 @@ import io.spine.protodata.java.render.JavaRenderer
  * Obtains all the messages known to the current compilation process
  * along with the corresponding file headers.
  */
-internal fun JavaRenderer.findMessageTypes(): Set<MessageWithFile> {
-    return select(ProtobufSourceFile::class.java).all()
+internal fun JavaRenderer.findMessageTypes(): Set<MessageWithFile> =
```

Comment:
Doesn't look like the content of this file has something to do with `validation`. I suggest moving it to `JavaRenderer` class.